data/coalition/coalition missions.txt

```diff
@@ -1684,3 +1684,184 @@ mission "Coalition: Alpha Encounter"
+					goto "know"
+				`	"No, the only aliens humans know about are the Quarg."`
+					goto "sad"
+			label sad
+			`	At this several in the audience sound disappointed.`
+				goto nextquestion
+			label know
+			`	Several members of the audience are delighted. "Cooperation between different species, it is important to foster," says the Saryd who asked the question. An Arach asks you to point out where the Hai are on a galactic map, and you do so.`
+			label nextquestion
+			branch war
+				has "FWC End: completed"
+			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
+			choice
+				`	"No, human space is split between the Republic in the north and the Free Worlds in the south."`
+				`	"No, human space is split between the Republic in the west, the Free Worlds in the south, and the Syndicate in the east. The Syndicate is technically part of the Republic, but they have a lot of autonomy."`
+			`	There are murmurs in the crowd, and several begin to write down notes on various devices."`
+				goto questionthree
+			label war
+			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
+			choice
+				`	"Yes, all of human space is united under the Republic."`
+				`	"Yes, but there are internal subdivisions: the Free Worlds in the south, the Republic in the north, and the Syndicate in the east."`
+				`	"No. Technically all of human space is under the Republic, but the Free Worlds and Syndicate have significant autonomy."`
+			`	There are murmurs in the crowd, and several begin to write down notes on various devices."`
+			label questionthree
+			`	The Arach picks another Kimek. "Still a problem, is piracy in human space?"`
+			choice
+				`	"Yes, there are pirate planets on the fringes of human space."`
+				`	"Piracy was a big problem in the past, but now it's a relatively minor issue."`
+			`	"Stamped out, may human piracy soon be," says the Arach in a solemn tone of voice. She picks a Saryd for the next question.`
+			`	"A fairly comprehensive map of human space, we have." The Saryd brings out a star map of the galactic south. "But of the core, we know not. Inform us of the galaxy, can you?"`
+			branch korath
+				or
+					has "First Contact: Korathi Efreti: offered"
+					has "First Contact: Laki Nemparu: offered"
+					has "First Contact: Karek Fornati: offered"
+					has "First Contact: Setar Fort: offered"
+					has "Wanderers: Rek To Kor Efret: offered"
+			choice
+				`	"Sorry, I don't know what's there any more than you do."`
+			`	The Saryd walks away, disappointed.`
+				goto sorry
+			label korath
+			choice
+				`	(Tell them about the Korath.)`
+					goto tell
+				`	(Don't tell them about the Korath.)`
+					goto donttell
+			label donttell
+			`	"Sorry, I don't know what's there any more than you do."`
+			`	The Saryd walks away, disappointed.`
```

Comment:
In Discord Xylily points out this seems to be missing a `goto sorry`.

Reply:
@roadrunner56 heads up that I posted https://github.com/endless-sky/endless-sky/pull/6211 if that's easiest

Reply:
Thanks for catching that, sorry for not responding until now.